Partially revert https://cgit.freedesktop.org/mesa/mesa/commit/?id=f7d42ee7d319
in order to allow ports like devel/allegro, devel/sdl{12,20}, graphics/glew,
graphics/libepoxy, x11-toolkits/qt5-gui to skip <GL/glext.h> inclusion via <GL/gl.h>.

--- include/GLES2/gl2.h.orig	2018-11-09 11:03:00 UTC
+++ include/GLES2/gl2.h
@@ -1,5 +1,5 @@
-#ifndef __gles2_gl2_h_
-#define __gles2_gl2_h_ 1
+#ifndef __gl2_h_
+#define __gl2_h_ 1
 
 #ifdef __cplusplus
 extern "C" {
--- include/GLES2/gl2ext.h.orig	2018-11-09 11:03:00 UTC
+++ include/GLES2/gl2ext.h
@@ -1,5 +1,5 @@
-#ifndef __gles2_gl2ext_h_
-#define __gles2_gl2ext_h_ 1
+#ifndef __gl2ext_h_
+#define __gl2ext_h_ 1
 
 #ifdef __cplusplus
 extern "C" {
--- include/GLES3/gl3.h.orig	2018-11-09 11:03:00 UTC
+++ include/GLES3/gl3.h
@@ -1,5 +1,5 @@
-#ifndef __gles2_gl3_h_
-#define __gles2_gl3_h_ 1
+#ifndef __gl3_h_
+#define __gl3_h_ 1
 
 #ifdef __cplusplus
 extern "C" {
--- include/GL/glcorearb.h.orig	2018-11-09 11:03:00 UTC
+++ include/GL/glcorearb.h
@@ -1,5 +1,5 @@
-#ifndef __gl_glcorearb_h_
-#define __gl_glcorearb_h_ 1
+#ifndef __glcorearb_h_
+#define __glcorearb_h_ 1
 
 #ifdef __cplusplus
 extern "C" {
--- include/GL/glext.h.orig	2018-11-09 11:03:00 UTC
+++ include/GL/glext.h
@@ -1,5 +1,5 @@
-#ifndef __gl_glext_h_
-#define __gl_glext_h_ 1
+#ifndef __glext_h_
+#define __glext_h_ 1
 
 #ifdef __cplusplus
 extern "C" {
--- include/GL/glxext.h.orig	2018-11-09 11:03:00 UTC
+++ include/GL/glxext.h
@@ -1,5 +1,5 @@
-#ifndef __glx_glxext_h_
-#define __glx_glxext_h_ 1
+#ifndef __glxext_h_
+#define __glxext_h_ 1
 
 #ifdef __cplusplus
 extern "C" {
